src/endpoints/endpoints.ts

```diff
@@ -1,13 +1,75 @@
-import express, { Express, Request, Response } from 'express';
-import { addAlice } from '../db/db';
+import { Express, Request, Response } from 'express';
+import { createUsers, getUserByID, getAllUsers,  updateUserByID } from '../db/db';
+import { deletUsersById } from '../db/db';
+
 
 export function endpoints(app: Express) {
+  app.post('/addUser', (req: Request, res: Response) => {
```

Comment:
i would suggest a rename of the end point to users
```suggestion
  app.post('/users', (req: Request, res: Response) => {
```

Reply:
since its a post i think the add part is implied